src/builder/QuarkBuilder.sol

```diff
@@ -16,16 +16,14 @@ contract QuarkBuilder {
     /* ===== Constants ===== */
 
     string constant VERSION = "1.0.0";
-    uint256 constant MAX_BRIDGE_ACTION = 1;
 
     /* ===== Custom Errors ===== */
 
     error AssetPositionNotFound();
-    error FundsUnavailable(uint256 requiredAmount, uint256 actualAmount, uint256 missingAmount);
+    error FundsUnavailable(string assetSymbol, uint256 requiredAmount, uint256 actualAmount, uint256 missingAmount);
     error InsufficientFunds(uint256 requiredAmount, uint256 actualAmount);
     error InvalidInput();
     error MaxCostTooHigh();
-    error TooManyBridgeOperations();
```

Comment:
So we're dropping `TooManyBridgeOperations()` check?

Reply:
Yeah, we discussed removing this earlier in the week. It's quite arbitrary and isn't really useful right now given there is only one supported bridge.